File: Gestion_Evenement2/src/main/java/edu/esprit/entities/Quiz.java
package edu.esprit.entities;

public class Quiz {
    private int idQz;
    private String name;
    private String type;
    private String image;

    public Quiz(){idQz=0;}
    public Quiz(int idQz, String name, String type, String image) {
        this.idQz = idQz;
        this.name = name;
        this.type = type;
        this.image = image;
    }

    public int getIdQz() {
        return idQz;
    }

    public void setIdQz(int idQz) {
        this.idQz = idQz;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }
}

File: projettest-main2/src/Entity/Quiz.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

use App\Repository\QuizRepository;

#[ORM\Entity(repositoryClass: QuizRepository::class)]

class Quiz
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private ?int $idqz = null;


    #[ORM\Column(length: 255)]
    private $type;

    #[ORM\Column(length: 255)]
    private $name;

    #[ORM\Column(length: 255)]
    private $image;

    public function getIdqz(): ?int
    {
        return $this->idqz;
    }

    public function getType(): ?string
    {
        return $this->type;
    }

    public function setType(?string $type): static
    {
        $this->type = $type;

        return $this;
    }

    public function getName(): ?string
    {
        return $this->name;
    }

    public function setName(string $name): static
    {
        $this->name = $name;

        return $this;
    }

    public function getImage(): ?string
    {
        return $this->image;
    }

    public function setImage(string $image): static
    {
        $this->image = $image;

        return $this;
    }


}